.travis/config.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
I guess it's not an issue, but this commented out env vars for runshellcheck.sh is missing.
```
--- template/.travis/config.sh	2020-08-03 15:12:27.504528495 -0700
+++ timesync/.travis/config.sh	2020-08-05 15:32:55.969311807 -0700
@@ -43,13 +43,8 @@
 #
 #   * .travis/runsyspycmd.sh:
 #
 #       - function lsr_runsyspycmd_hook
 #
 #   * .travis/runpytest.sh:
 #
 #       - RUN_PYTEST_SETUP_MODULE_UTILS
-#
-#   * .travis/runshellcheck.sh:
-#
-#       - RUN_SHELLCHECK_DISABLED
-#       - RUN_SHELLCHECK_EXTRA_ARGS

```